index.html

```diff
@@ -8,7 +8,7 @@
   <title>Camilo Vela Portfolio</title>
   <link rel="preconnect" href="https://fonts.googleapis.com">
   <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-  <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@400;700&family=Poppins&family=Roboto&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@400;500;700&family=Poppins&family=Roboto&display=swap" rel="stylesheet">
 </head>
 <body>
   <section class="container">

```

Comment:
You have not changed your navbar for the desktop screen. Kindly do so.
| Your design  | Figma design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/33268927/177907527-b3d23c22-c275-4c0f-962a-ce6891881bd1.png) | ![image](https://user-images.githubusercontent.com/33268927/177907597-d72b5ba2-0603-4b81-9b2c-ba5b536579d7.png) |